teensy_loader_cli.c

```diff
@@ -243,6 +243,42 @@ usb_dev_handle * open_usb_device(int vid, int pid)
 				}
 			}
 			#endif
+
```

Comment:
Note, this block needs to be above the ifdef LIBUSB_HAS_GET_DRIVER_NP block. Otherwise devices that are not selected (because the serial number doesn't match) will be detached from the kernel and not added back. We should only detach the kernel driver if we're actually going to update the device.